ferrocene/ci/scripts/emulated-x86_64-qnx-test-runner.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT OR Apache-2.0
+# SPDX-FileCopyrightText: The Ferrocene Developers
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# qcc resides in `somewhere/qnx710/host/linux/x86_64/usr/bin`
+qnxdir="$(realpath $(dirname $(which qcc))/../../../../..)"
+emulatordir=/tmp/emulator
```

Comment:
If this directory doesn't exist, the script reports an error until you `mkdir` it.

Reply:
yes, that's intentional. the presence of that directory is explicitly checked in `cmd_prepare`.

this script should not create the directory because it's mounted by the host before this script runs inside a docker container in CI. creating the directory, e.g. with `mkdir -p`, would mask an error in CI where the directory was not mounted by the host.

not creating the directory is also in line with how the pre-existing `emulated-aarch64-test-runner.sh` (note: not the QNX one) script behaves